### PR TITLE
Updated Collations and Case Sensitivity

### PR DESCRIPTION
**Pull request by jessegood**
Made it clear the default collation is OS locale dependent, also a few other minor fixes.

### CODE REVIEW

File: entity-framework/core/miscellaneous/collations-and-case-sensitivity.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ Text processing in databases can be complex, and requires more user attention th
...
+A fundamental concept in text processing is the *collation*, which is a set of rules determining how text values are ordered and compared for equality. For example, while a case-insensitive collation disregards differences between upper- and lower-case letters for the purposes of equality comparison, a case-sensitive collation does not. However, since case-sensitivity is culture-sensitive (e.g. `i` and `I` represent different letters in Turkish), multiple case-insensitive collations exist, each with there own set of rules. The scope of collations also extends beyond case-sensitivity, to other aspects of character data; in German, for example, it is sometimes (but not always) desirable to treat `ä` and `ae` as identical. Finally, collations also define how text values are *ordered*: while German places `ä` after `a`, Swedish places it at the end of the alphabet.
```
Comment (roji) [inline entity-framework/core/miscellaneous/collations-and-case-sensitivity.md:17]: "there" is incorrect - you probably meant "their". I believe the current "its" is better.

The other fix is stylistic ("there exist" is fine), I'd rather we kept it as it is.

File: entity-framework/core/miscellaneous/collations-and-case-sensitivity.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ Text processing in databases can be complex, and requires more user attention th
...
 
-All text operations in a database use a collation - whether explicitly or implicitly - to determine how the operation compares and orders strings. The actual list of available collations and their naming schemes is database-specific; consult [the section below](#database-specific-information) for links to relevant documentation pages of various databases. Fortunately, database do generally allow a default collation to be defined at the database or column level, and to explicitly specify which collation should be use for specific operations in a query.
+All text operations in a database use a collation - whether explicitly or implicitly - to determine how the operation compares and orders strings. The actual list of available collations and their naming schemes is database-specific; consult [the section below](#database-specific-information) for links to relevant documentation pages of various databases. Fortunately, databases generally allow a default collation to be defined at the database or column level, and to explicitly specify which collation should be used for specific operations in a query.
```
Comment (roji) [inline entity-framework/core/miscellaneous/collations-and-case-sensitivity.md:19]: Please leave the "do" in.

File: entity-framework/core/miscellaneous/collations-and-case-sensitivity.md [hunk 3]
```diff
@@ -14,13 +14,13 @@ Text processing in databases can be complex, and requires more user attention th
...
+In most database systems, a default collation is defined at the database level; unless overridden, that collation implicitly applies to all text operations occurring within that database. The database collation is typically set at database creation time (via the `CREATE DATABASE` DDL statement), and if not specified, defaults to a some server-level value determined at setup time. For example, the default server-level collation in SQL Server for the "English (United States)" locale is `SQL_Latin1_General_CP1_CI_AS`, which is a case-insensitive, accent-sensitive collation. Although database systems usually do permit altering the collation of an existing database, doing so can lead to complications; it is recommended to pick a collation before database creation.
```
Comment (roji) [inline entity-framework/core/miscellaneous/collations-and-case-sensitivity.md:23]:
```suggestion
In most database systems, a default collation is defined at the database level; unless overridden, that collation implicitly applies to all text operations occurring within that database. The database collation is typically set at database creation time (via the `CREATE DATABASE` DDL statement), and if not specified, defaults to a some server-level value determined at setup time. For example, the default server-level collation in SQL Server for the "English (United States)" machine locale is `SQL_Latin1_General_CP1_CI_AS`, which is a case-insensitive, accent-sensitive collation. Although database systems usually do permit altering the collation of an existing database, doing so can lead to complications; it is recommended to pick a collation before database creation.
```

**Review by roji — CHANGES_REQUESTED** [review]

File: entity-framework/core/miscellaneous/collations-and-case-sensitivity.md (hunk 1 above)
Comment (jessegood) [reply entity-framework/core/miscellaneous/collations-and-case-sensitivity.md]: Thanks. Yes I meant "their". I thought it read easier, but decided to leave it as-is.

**Review by roji — APPROVED** [review]
Thanks!

**Pull request merged**